conman_ros/src/ros_interface_service.cpp

```diff
@@ -34,10 +34,13 @@ ROSInterfaceService::ROSInterfaceService(RTT::TaskContext* owner) :
   }
 
   // Connect operation callers
-  RTT::log(RTT::Debug) << "Connecting conamn_ros operation callers..." << RTT::endlog();
-  getBlocks = scheme->getOperation("getBlocks");
-  getGroups = scheme->getOperation("getGroups");
-  switchBlocks = scheme->getOperation("switchBlocks");
+  RTT::log(RTT::Debug) << "Connecting conamn_ros operation callers..." << RTT::endlog(); 
+  getBlocks = RTT::OperationCaller<bool(std::vector<std::string>&, std::vector<std::string>&, bool, bool)>(
+      scheme->getOperation("getBlocks"), scheme->engine());
+  getGroups = RTT::OperationCaller<bool(std::vector<std::string>&, std::vector<std::string>&, bool, bool)>(
+      scheme->getOperation("getGroups"), scheme->engine()); 
```

Comment:
The signature for this `OperationCaller` isn't right, it needs to be the same signature as the `Scheme::getGroups` function.